Sources/StreamVideoSwiftUI/Utils/PaddingsConfig.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A configuration object that defines padding values for different edges.
+///
+/// `PaddingsConfig` allows you to specify custom padding values for leading,
+/// trailing, top, and bottom edges, which can be reused across multiple UI components.
+public struct PaddingsConfig {
```

Comment:
Why not use the EdgeInsets struct which is already defined?

Reply:
What is the difference that gets you on the fence. They look quite similar to me.

Reply:
As a bonus, if you change it to EdgeInsets you don't need to implement the modifier as the existing `.padding` modifier handles `EdgeInsets` already

Reply:
updated, looks much nicer now